mlflow/metrics/__init__.py

```diff
@@ -191,6 +197,53 @@
 .. _rougeLsum: https://huggingface.co/spaces/evaluate-metric/rouge
 """
 
+
+# Regression Metrics
+#TODO: how to deal with sample weights input?
+#TODO: double check greater is better
+
+mae = make_metric(
+    eval_fn=_mae_eval_fn,
+    greater_is_better=False,
+    name="mae",
+    version="v1"
```

Comment:
Sorry, forgot to mention this, we don't actually want to put a version on these metrics since we don't want to change the names of the metrics for backwards compatibility (with version the name will become mae/v1 instead of mae which it is now)